kolibri/utils/server.py

```diff
@@ -913,77 +914,116 @@ def get_urls(listen_port=None):
         return e.status_code, []
 
 
+def get_installer_version(installer_type):  # noqa: C901
+    def get_debian_pkg_version(package):
+        """
+        In case we want to distinguish between dpkg and apt installations
+        we can use apt-cache show madison and compare versions with dpkg
+        if dpkg > madison, it's dpkg otherwise it's apt
+        """
+        try:
+            output = check_output(["dpkg", "-s", package])
+            if hasattr(output, "decode"):  # needed in python 2.x
+                output = output.decode("utf-8")
+            package_info = output.split("\n")
+            version_info = [output for output in package_info if "Version" in output]
```

Comment:
Not sure why this line didn't raise an issue before, but it's now causing a python 2 syntax check flake8 failure:

```
kolibri/utils/server.py:929:40: F812 list comprehension redefines 'output' from line 927
```
https://github.com/indirectlylit/kolibri/runs/4301449205?check_suite_focus=true

Perhaps this should be:

```python
             version_info = [info for info in package_info if "Version" in info]
```

Weirdly, I see the linting error in my fork but not in the LE org repo.

cc @jredrejo @rtibbles

Reply:
interesting, the check is skipped in LE but not in my repo:

![image](https://user-images.githubusercontent.com/2367265/143060249-7f81f5e1-032f-48f4-91ef-243725b4a30e.png)

Reply:
Oh - probably because the path match check looks back through the history of runs (which are constrained to a specific fork) so because the learning equality repo had a test run that had previously passed for this set of files, it doesn't run again, whereas your fork didn't.

Reply:
filed a follow-up here: https://github.com/learningequality/kolibri/issues/8772

Reply:
weord, that's not new code, it has been there since kolibri 0.11

Reply:
I think it is more likely a floating linter version issue than an issue with the CI setup.